pwa/src/hooks/object.ts

```diff
@@ -31,6 +32,27 @@ export const useObject = () => {
       enabled: !!objectId && !isDeleted(objectId),
     });
 
+  const downloadPDF = () =>
+    useMutation<any, Error, any>(API.Object.downloadPDF, {
+      onSuccess: async (data, variables) => {
+        const url = window.URL.createObjectURL(new Blob([data]));
+        const link = document.createElement("a");
+        link.href = url;
+
+        link.setAttribute("download", `${variables.name}.pdf`);
+        document.body.appendChild(link);
+        link.click();
```

Comment:
I'm thinking we could move this logic to a service, so that we can easily re-use it, something like this:

```
export const downloadBlob = (data: any, name: string): void => {
  const url = window.URL.createObjectURL(new Blob([data]));
  const link = document.createElement("a");
  link.href = url;

  link.setAttribute("download", `${name}.pdf`);
  document.body.appendChild(link);
  link.click();
};

```